homeassistant/components/homekit/__init__.py

```diff
@@ -361,9 +361,17 @@ def add_bridge_accessory(self, state):
             return
         aid = generate_aid(state.entity_id)
         conf = self._config.pop(state.entity_id, {})
-        acc = get_accessory(self.hass, self.driver, state, aid, conf)
-        if acc is not None:
-            self.bridge.add_accessory(acc)
+        # If an accessory cannot be created or added due to an exception
+        # of any kind (usually in pyhap) it should not prevent
+        # the rest of the accessories from being created
+        try:
+            acc = get_accessory(self.hass, self.driver, state, aid, conf)
+            if acc is not None:
+                self.bridge.add_accessory(acc)
+        except Exception:  # pylint: disable=broad-except
```

Comment:
Would it be possible to catch more specific exceptions? (and then log with `_LOGGER.error`)

Also, would it be possible to retry this accessory or are these all permanent errors?

Reply:
Its anything the `pyhap` codebase can throw.  The most common is `ValueError` but its a wide mix so I went with logging it.

All errors are likely to be permanent.